src/tests/Microsoft.Diagnostics.Tools.RuntimeClient/eventpipetests.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Can we avoid making this change? We'd like for the tool process not to require a new runtime even if the target being traced does require 3.0.

Reply:
This is the test project. It connects to itself (using IPC), and it will only work in `netcoreapp3.0`.

Reply:
I mistakenly made it 2.1 in previous PR because VS2017 wouldn't build `netcoreapp3.0` by default.

Reply:
Ah, my misread. No qualms if test app is 3.0, I only care about RuntimeClient library. Thanks!